css/style.css

```diff
@@ -0,0 +1,199 @@
+  padding: 5px;
+  z-index: 2;
+  color: white;
+  position: absolute;
+}
+
+.heading {
+  margin-bottom: 25px;
+}
+
+footer p {
+  clear: both;
+  width: 49%;
+  font-size: 11px;
+  padding-top: 1.5em;
+  text-align: left;
+  margin-left: 200px;
+  color: #6e6e6e;
+}
+
+.heading a {
+  color: white !important;
+  margin-top: 200px;
+}
+
+a.play {
+  color: white;
+  font-size: 25px;
+  font-weight: 300;
+  font-family: 'Lato', Helvetica, sans-serif;
+  margin-top: 100px !important;
+}
+
+.play:hover {
+  text-decoration: underline;
+}
+
+.navbar {
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))), url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA9QAAAAkCAMAAABfcIIyAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAHVQTFRFAAAA%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F5YtmQAAAACd0Uk5TAAECAwQFBgcICQoLDA0ODxAREhMUFRYXGBkaGxwdHh8gISIjJCUmZiD6WAAAAdlJREFUeF7t3btuFEEQheFzunotGYkIERA4spn3fyeHiAgEEtsXB82s2eEJavR%2F0Wjyo75XWfHpw4MkSXNqCkA2lr2%2B%2Fvz63n15qpI0x5yTSAM52XaxJLVXf%2FkoaY6xQk2sgXy8Ql2KJf3wS5HG6HMMQg3kZNkuxVGKNLxJo48%2BxhxTpBrIx7JcXEqUKJI3zd77CjWLaiAle4U6Iixv6q33MfocTL%2BBlCy7OEqJqCFvs%2FV2m3%2BTaiAd6zb7rlHtrbc1VA%2Bm30BSa%2B%2B7RNSo4a21thbVTL%2BBnCy7rCV1rdXbtbXW%2B%2Bh%2Ft78BZLM2v6NE1Fov%2FrpC3Zl%2BA2l5H6lrrRe%2FXNu%2BqCbUQE57qGvsoW5t9D4H028gJculOKLUWuvFz%2B16O9NipAZSsm8nWpfq53ZlTQ3k9s%2BamlADZ0CogZMh1MDJEGrgZAg1cDL%2Fh5ojLSC1%2ByMtLp8A2R0un3BNFEjv%2FpooDzqA9O4fdPD0Esju8PSSIglAdsciCZQzArI7lDOat30yCg8CGe2FB1eovZcI7oMSwUBSq0TwWlQHxfyBEzgW86ftDpDase0ODfKA7A4N8mhlC2R318r2px%2BeQqLpPJDae9P5%2FmrVz491%2FWekBlKy7PXVfn9rb9xXsMD2wdOcAAAAAElFTkSuQmCC') no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  border-radius: 4px;
+  height: 34px;
+  margin: 100px auto 0;
+  width: 980px;
+  z-index: 999;
+}
+
+.end {
+  margin-bottom: 30px;
+}
+
+.list {
+  font-size: 0.85rem;
+  border-right: 1px solid grey;
+  border-left: 1px solid #505050;
+  padding-top: 8px;
+  width: 11.111%;
+}
+
+.list:first-of-type {
+  padding-top: 4px;
+  border-left: none;
+}
+
+.list:last-of-type {
+  padding-top: 6px;
+  border-right: none;
+}
+
+.list:hover {
+  background: url('../img/hover.jpg') 0 0 repeat;
+  border-radius: 4px;
+  padding: 10px;
```

Comment:
#### `REQUIRED`

Why is there `padding` of `10px` here? :thinking: 
Here's the current effect of that... 

| Current | Preffred |
| :-----: | :-----: |
| ![clone apple nav hover](https://user-images.githubusercontent.com/36057474/63677213-6f94bb80-c7e4-11e9-8b90-e20b031ddefd.gif) | ![preffered apple nav hover](https://user-images.githubusercontent.com/36057474/63677017-0d3bbb00-c7e4-11e9-8c50-ffae0f3fd5ca.gif) |

### Consider removing it. :relaxed: